docs/readmes/howtos/inbound_roaming.md

```diff
@@ -40,8 +44,7 @@ Configurations with SubscriberDB, local HSS and remote HSS all at the same
 time is not supported yet.
 
 ## Prerequisites
-
-Before starting to configure roaming setups, first you need to bring up a
+Before starting to configure roaming setup, first you need to bring up a
 setup to handle your own/local subscribers.
 
 Even one of the architectures mentions `Non-Federated`, Inbound Roaming
```

Comment:
One possible re-write: 

> In both the architectures mentioned above, the Inbound Roaming feature requires a federation gateway to > create the S6a and S8 interfaces with the remote HSS and PGW respectively. To configure Inbound  Roaming, the pre-requisites are: 
>
> * An Orc8r 
> * A `feg` network and a FeG (For roaming subscribers)
> * A `feg_lte` network and an AGW (Access for local & roaming subscribers)

Reply:
I see this paragraph was unfortunate. My idea was to instruct the reader that before configuring Inbound Roaming, they need to go through the other documents and make sure that their local setup works.